eCommerce/users/management/commands/create_superuser.py

```
@@ -0,0 +1,24 @@
+import os
+from django.core.management.base import BaseCommand
+from users.models import SiteUser
+
+class Command(BaseCommand):
+    """
+    Custom management command to create a superuser from environment variables.
+    """
+    help = 'Creates a superuser using credentials from environment variables'
+
+    def handle(self, *args, **options):
```

Ruff (0.14.6)

Unused method argument: `args`

(ARG002)